cmake/modules/hunter_config.cmake

```diff
@@ -8,6 +8,61 @@ include(hunter_pack_git_submodule)
 include(hunter_unsetvar)
 include(hunter_user_error)
 
+function(parse_cmake_args_for_keyword cmake_args)
```

Comment:
ok, I'll add a new file `hunter_parse_cmake_args_for_keyword.cmake` or would you prefer another name?